Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Email-Address/index.md

```diff
@@ -0,0 +1,49 @@
+---
+versionFrom: 8.0.0
+---
+
+# Dropdown
+
+`Alias: Umbraco.DropDown.Flexible`
+
+`Returns: String`
+
+Displays an email address.
+
+## Settings
+
+### Enable required checkbox (8.7.0 and below)
+
+If you type an email thats not correct the Umbraco will display a warning label under the property editor. When you publish the page that this editor is located on, Umbraco will clean the input.
```

Comment:
I'm not entirely sure I understand what is meant by this part here.
Could you perhaps elaborate a bit, and try to make a little clearer?

```suggestion
If you type an email that's not correct Umbraco will display a warning label under the property editor. When you publish the page that this editor is located on, Umbraco will clean the input.
```